libcaf_core/caf/actor_system_config.cpp

```diff
@@ -488,4 +492,13 @@ const settings& content(const actor_system_config& cfg) {
   return cfg.content;
 }
 
+// -- factories ----------------------------------------------------------------
+
+intrusive_ptr<logger>
+actor_system_config::make_logger(actor_system& sys) const {
+  if (logger_factory_)
```

Comment:
Would it be better if we use `has_logger_factory` here instead?

Reply:
We call `logger_factory_(sys)` on the next line, so I think using the member function here would do more obfuscation than good.